CONTRIBUTE.md

```diff
@@ -13,9 +12,11 @@ https://learn.microsoft.com/en-gb/ef/core/cli/dotnet
 
 Das hab ich direkt in Rider als Paket (Nu Get) hinzugefügt (War einfacher).
 
+```
 dotnet add package Microsoft.EntityFrameworkCore.Design
+```
 
-### Preperation on M1 Mac
+### Vorbereitungen für Apple Silicon Macs
 
 Zu Beginn hatte ich Probleme dotnet ef auszuführen. Nach einiger Recherge bin ich darauf gestossen:
```

Comment:
Recherche

Reply:
Having a problem with this word. Always. 🤪